src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -23,6 +23,19 @@ public class ViewLoadedParams : ReadyParams
         private readonly DynamoViewModel dynamoViewModel;
         public readonly Menu dynamoMenu;
 
+        /// <summary>
+        /// A reference to the <see cref="ViewStartupParams"/> class.
+        /// Useful if this extension will be loaded from a package as its startup method, will not be called.
+        /// </summary>
+        public ViewStartupParams ViewStartupParameters
+        {
+            get
+            {
+                var startupParams = new ViewStartupParams(this.dynamoViewModel);
+                return startupParams;
```

Comment:
I would recommend making this simply a `private readonly` field with a `property`. Also, why not name this the same as the type name - `ViewStartupParams`?

Reply:
will do.